<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Germanium Selector Builder v2.0.5</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="browserLabel">
          <property name="text">
           <string>Browser:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="browserCombo"/>
        </item>
        <item>
         <widget class="QPushButton" name="startBrowserButton">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopBrowserButton">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="liveButton">
          <property name="text">
           <string>Live</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="highlightElementButton">
          <property name="text">
           <string>Highlight</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pickElementButton">
          <property name="text">
           <string>Pick</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelPickButton">
          <property name="text">
           <string>Cancel Pick</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="codeEdit">
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionAbout_QT"/>
   </widget>
   <widget class="QMenu" name="menuExecute">
    <property name="title">
     <string>Execute</string>
    </property>
    <addaction name="actionHighlight"/>
    <addaction name="actionPick"/>
   </widget>
   <addaction name="menuExecute"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_About">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHighlight">
   <property name="text">
    <string>Highlight</string>
   </property>
  </action>
  <action name="actionPick">
   <property name="text">
    <string>Pick</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>About QT</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
